Gemfile.lock

```diff
@@ -42,7 +42,7 @@ GEM
     jwt_signed_request (2.4.0)
       jwt (~> 1.5.0)
       rack
-    loofah (2.2.2)
+    loofah (2.2.3)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.2.2 → 2.2.3

[_~~change-log~~, [source-code](http://github.com/flavorjones/loofah)_]

:tada: Patched vulnerabilities:

- [CVE-2018-16468](https://nvd.nist.gov/vuln/detail/CVE-2018-16468)
  **Loofah XSS Vulnerability**

  
  URL: https://github.com/flavorjones/loofah/issues/154

  In the Loofah gem, through v2.2.2, unsanitized JavaScript may occur in sanitized output when a crafted SVG element is republished.






<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.2.2...v2.2.3).

These are the individual commits:
- (de6b0f3) [extract msword html data into an asset file](https://github.com/flavorjones/loofah/commit/de6b0f33cde92b6028c1ef973e5fc24478890fc9)
- (ac7c50d) [updated mailing list to a new Google Group](https://github.com/flavorjones/loofah/commit/ac7c50de12398c90ffba907bf132af66bcc242be)
- (3556e2b) [add formatting to CHANGELOG](https://github.com/flavorjones/loofah/commit/3556e2b44f7401aaccbb10e2abac4e044391267a)
- (71e4b54) [remove the svg animate attribute `from` from the allowlist](https://github.com/flavorjones/loofah/commit/71e4b5434fbcb2ad87643f0c9fecfc3a847943c4)
- (cb3dbfa) [version bump to v2.2.3 and update CHANGELOG](https://github.com/flavorjones/loofah/commit/cb3dbfa604195b99b3a811e040584daec7663504)


</details>